app/scenes/Settings/components/GroupListItem.js

```diff
@@ -33,15 +35,20 @@ class GroupListItem extends React.Component<Props> {
   onEdit = () => {};
 
   render() {
-    const { group, showMenu } = this.props;
+    const { group, groupMemberships, showMenu } = this.props;
+    const membersCount = groupMemberships.inGroup(group.id).length;
 
     return (
       <React.Fragment>
         <ListItem
           title={
             <Title onClick={this.handleMembersModalOpen}>{group.name}</Title>
           }
-          subtitle={<React.Fragment>10 members</React.Fragment>}
+          subtitle={
+            <React.Fragment>
+              {membersCount} member{membersCount === 1 ? '' : 's'}
+            </React.Fragment>
```

Comment:
I didn't see a pluralizer lib, is there one?

Reply:
Weirdly, there isn't – I've somehow gotten away without it.